phpstan.neon

```diff
@@ -0,0 +1,30 @@
+parameters:
+	level: 3
+	paths:
+		- src
+		- tests
+	ignoreErrors:
+		-
+			message: '#Call to an undefined method Iterator\:\:append\(\)#'
+			paths:
+				- src/Connector/V1
```

Comment:
This PR also checks classes from the V1 namespace. The V1 classes will be removed in this PR: #36  It will be wise to check this PR - where the V1 classes are removed - first, review it and approve it. This way the V1 classes are removed and these ignoreErrors can be removed from this PR.